eslint.config.mjs

```diff
@@ -0,0 +1,11 @@
+import globals from "globals";
+import pluginJs from "@eslint/js";
+import tseslint from "typescript-eslint";
+
+
+export default [
+  {files: ["**/*.{js,mjs,cjs,ts}"]},
+  {languageOptions: { globals: globals.browser }},
+  pluginJs.configs.recommended,
+  ...tseslint.configs.recommended,
```

Comment:
It seems like you're trying to spread the recommended configs from 'pluginJs' and 'tseslint' into the array, but this is not the correct way to extend configurations in ESLint. You should use the 'extends' property within a configuration object.